src/cozy/react/components/StepWrapper.jsx

```diff
@@ -8,10 +8,12 @@ import { STEPS } from 'cozy/react/steps'
 
 const COZY_BAR_HEIGHT = 48
 
+const ROUTES_WITH_DEFAULT_BAR_LEFT = [STEPS.PRESENTATION, STEPS.INSTALLATION]
```

Comment:
Shouldn't this be name `WITHOUT` instead of `WITH`?

Or maybe by `DEFAULT_BAR_LEFT` you mean `the default icon on the left side of bar` and not the `left arrow`?

Reply:
Exactly "the default icon on the left side of bar"

Reply:
I updated to "ROUTES_WITHOUT_BACK_BUTTON"